packages/api/src/api/appConfig.ts

```diff
@@ -0,0 +1,72 @@
+import { Router } from 'express';
+import logger from '../logger';
+import { AppConfig } from '../entities/AppConfig';
+import { populateUser } from '../middleware/populateUser';
+
+export const appConfig = Router();
+appConfig.use(populateUser());
+
+appConfig.get('/', async (req, res) => {
+  const user = req.userEntity;
+  if (!user.isAdmin) {
+    res.status(403).send('only admins have permission to access this endpoint!');
+    return;
+  }
+
+  try {
+    const configItems = await req.entityManager.find(AppConfig, {}, { orderBy: { key: 'ASC' } });
+    res.status(200).send(configItems);
+  } catch (error) {
+    const errorMsg = 'There was an issue fetching the list of config items';
+    logger.error(`${errorMsg}: `, error);
+    res.status(500).send(errorMsg);
+  }
+});
+
+appConfig.put('/', async (req, res) => {
+  const user = req.userEntity;
+  const { key, value } = req.body;
+
+  if (!user.isAdmin) {
+    res.status(403).send('only admins have permission to access this endpoint!');
+    return;
+  }
+
+  try {
+    const configItemToUpdate = await req.entityManager.findOne(AppConfig, { key });
+
+    if (!configItemToUpdate) {
+      res.status(400).send('The specified config item does not exist');
+      return;
+    }
+
+    configItemToUpdate.value = value;
+    await req.entityManager.persistAndFlush(configItemToUpdate);
```

Comment:
After making `key` unique, simply create a new `AppConfig` and try `persistAndFlush`